conf/modules.config

```diff
@@ -93,4 +93,21 @@ process {
         ]
     }
 
+    withName: 'GENERATE_IMAGES' {
+        ext.args = '--host http://localhost --ports 8000-8099 --timeout 60'
```

Comment:
I would suggest removing this line since the values there are the BTK defauts
```suggestion
```
cf
```
  --host STRING     Host name. [Default: http://localhost]
  --path STRING     Path to directory containing BlobDir. [Default: .]
  --ports RANGE     Range of ports available for hosting BlobTools Viewer/API. [Default: 8000-8099] = str(snakemake.params.ports)
  --timeout INT     Maximum time (in seconds) to wait for images. [Default: 30]
```